resources/js/requests/components/RequestsListing.vue

```diff
@@ -426,6 +427,13 @@ export default {
             },
           )
           .then((response) => {
+            if (response.data.data.length === 0 && this.fetchFlag === 0){
+              this.page = 1;
+              this.fetch();
+              this.fetchFlag = 1;
+            } else {
+              this.fetchFlag = 0;
+            }
```

Comment:
This solution is incomplete. Please, it is necessary to review the logic of the task list. It is most likely that the same behavior is occurring when you apply a filter, it is searching on the current page being viewed. It is necessary to check the different pages.

Reply:
@CarliPinell @gproly I agree. Please reset the page back to 1 when ever a filter changes

Reply:
Thank you very much Roly for the comments. However in 'Tasks' there is no problem with the filter on the column that contains date as Due Date.

I will review the files you tell me, for now this solution works well and solves the problem reported by QA.

I look forward to any suggestions or improvements that Nolan can give us. Unfortunately these errors are reported very late when there is not much time left to do a more in-depth analysis.

Reply:
Solved. Logic was changed.